### PR TITLE
Append querystring to package binary download to bust CDN cache

### PR DESCRIPTION
**Pull request by maartenba**
Partial fix for https://github.com/NuGet/NuGetGallery/issues/3168
Adds a query string parameter to the package binary download so we fetch
a fresh cache version from the CDN.

@scottbommarito @skofman1 @ryuyu

### CODE REVIEW

File: src/Ng/Feed2Catalog.cs [hunk 1]
```diff
@@ -304,7 +304,10 @@ static DateTime ForceUtc(DateTime date)
             {
                 foreach (var packageItem in entry.Value)
                 {
-                    var response = await client.GetAsync(packageItem.ContentUri, cancellationToken);
+                    var packageUriBuilder = new UriBuilder(packageItem.ContentUri);
```
Comment (skofman1) [inline src/Ng/Feed2Catalog.cs]: >packageUriBuilder [](start = 24, length = 17)

can u please add a comment explaining this change (for future generations)

**Comment by skofman1** [comment]
:shipit:

File: src/Ng/Feed2Catalog.cs (hunk 1 above)
Comment (maartenba) [inline src/Ng/Feed2Catalog.cs]: Added :-)

**Comment by scottbommarito** [comment]
LGTM!

**Comment by ryuyu** [comment]
LGTM

**Comment by shishirx34** [comment]
Looks Good.

**Pull request merged**